earthpy/plot.py

```diff
@@ -431,12 +431,14 @@ def hist(
         A list of title values that should either equal the number of bands
         or be empty. A string is accepted for a single dimension array.
     xlabel : str (optional)
-        The text to print on the x axis
+        The text to print on the x axis.
     ylabel : str (optional)
-        The text to print on the y axis
-    **kwargs : key, value pairings optional
-        These will be passed to the matplotlib .hist call
-        See https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.hist.html
+        The text to print on the y axis.
+    range : list (default = [np.nanmin(arr), np.nanmax(arr)])
```

Comment:
this should be a tuple.https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.axes.Axes.hist.html

Reply:
I borrowed some of the documentation from matplotlib to document the range parameter in EarthPy. Is there a way to cite them, or is that not common practice?

Reply:
i've seen people say: from the docs and provide a link? or you can just provide a link as i see was in the code previously?? does that help?